webview/src/experiments/components/Experiments.tsx

```diff
@@ -108,7 +112,9 @@ export const ExperimentsTable: React.FC = () => {
     toggleAllRowsExpanded()
   }, [toggleAllRowsExpanded])
 
-  const hasOnlyDefaultColumns = columns.length <= 1
+  const hasOnlyDefaultColumns = columns.every(
+    ({ id }) => id && isDefaultColumn(id)
```

Comment:
Updated this to check for the column ids since there are now 1 or 3 default columns depending on if the table is sorted.

Reply:
[Q] Does `every` return `true` if the array is empty?